### ISSUE TITLE
Very strange issue with Dark Souls III

### ISSUE THREAD

**Issue by rolus** [x360ce/x360ce#587]
For some reason x360ce completely stopped working with das3 after I reinstalled it, and in a very strange way as well: whenever I launch the 64 bit version x360ce doesn't detect my controller at all: http://i.imgur.com/0rcFLCc.png
Somehow this only happens when I try to launch it in the das3 folder; also, when I try to launch the 32 bit version it works fine (and doesn't even bring up the "this is 32 bit version of x360ce but the application is 64 bit" warning message), but obviously does nothing in-game as it's a 64 bit game.
Both versions work completely fine with other games, 64 or 32 bit.
I've worked around it by simply copying the x360ce files from another game to the das3 folder and it seems to work fine, but I'm really confused as to what causes the issue at all.

**Comment by Squall-Leonhart** [comment]
You cannot use the 64bit app with a 32bit xinput dll, or as vice versa.  the app can create the dlls for either arch type however.

**Comment by rolus** [comment]
Yes, I know that, the problem is that the app doesn't seem to attempt creating the dlls or detecting my gamepad when I launch it in a folder containing the das3 executable, while working as intended for everything else.

**Comment by EvoJocys** [comment]
This report is about the 3.x line, where x360ce was an xinput DLL copied into each game's folder. The current 4.x line works differently: it creates a virtual Xbox 360 controller that every game sees, with nothing copied into game folders. Game-specific problems from the DLL era do not carry over.

The latest release is 4.21 (September 2026): https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
